database/dynamodb/util.go

```diff
@@ -196,11 +196,13 @@ func (db *DynamoDB) tableIsAlive(ctx context.Context, name string) error {
 		case <-ctx.Done():
 			return ctx.Err()
 		case <-ticker:
-			r, _ := db.client.DescribeTable(&dynamodb.DescribeTableInput{
+			r, err := db.client.DescribeTable(&dynamodb.DescribeTableInput{
 				TableName: aws.String(name),
 			})
-			if *r.Table.TableStatus == "ACTIVE" {
-				return nil
+			if err == nil {
```

Comment:
Should we log this error? If it's a permissions error or something, this might wait silently forever.

Reply:
True... maybe it should check for a specific error type instead